include/io.tcc

```diff
@@ -28,24 +28,26 @@ void IO::write_vertices(
 }
 
 //! \brief Write initial stresses of material points
-//! \param[in[ stresses Initial stress of material points
-void IO::write_stresses(const std::vector<std::array<double, 6>>& stresses) {
+//! \param[in] stresses Initial stress of material points
+void IO::write_stresses(
+    const std::vector<std::shared_ptr<Point<3>>>& vertices) {
```

Comment:
If you are changing function arguments, you need to change the comments! Anyway, this should a vector of stresses.